plugins/restiming.js

```diff
@@ -1,5 +1,5 @@
 /**
-\file restiming.js
+file restiming.js
```

Comment:
This is part of Doxygen syntax (though we will be standardizing on JSDoc soon) -- please revert for now.